local/README.md

```diff
@@ -0,0 +1,44 @@
+# local development scripts
+A home for experimental scripts used for launching Nextflow Tower workflows
+
+## Setup
+
+These instructions assume that you already have Python, a Python version manager (`pyenv`), and `pipenv` installed.
+
+### Python Environment
+
+Set up your Python environment for using these scripts by ruinning
+```
+pipenv install --dev
+pipenv shell
+```
+
+### Environment Variables
+
+In order for the scripts leveraging `py-orca` to connect to Nextflow Tower, you will need to configure the following environment variables:
+    - `NEXTFLOWTOWER_CONNECTION_URI`
+    - `SYNAPSE_CONNECTION_URI`
+    - `AWS_ACCESS_KEY_ID`
+    - `AWS_SECRET_ACCESS_KEY`
+    - `AWS_SESSION_TOKEN`
+    * Optionally, you can use `aws sso login` in place of the AWS environment variable credentials if you have pre-configured AWS profiles already.
```

Comment:
In that case, you would use `AWS_PROFILE` correct?

Reply:
Right. I added `AWS_PROFILE` to the example `.env` file and added a description here.